spec/lib/tasks/evaluation_spec.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.describe "Evaluation tasks" do
+  describe "evaluation:clickstream:setup_sample_set" do
+    let(:sample_set) { class_double(DiscoveryEngine::Evaluation::SampleQuerySet) }
+    let(:sample_query_set_service_stub) { double("sample_query_set_service", create_sample_query_set: double(name: "name")) }
+    let(:sample_query_service_stub) { double("sample_query_service", import_sample_queries: double(wait_until_done!: true, error?: false)) }
+
+    before do
+      allow(DiscoveryEngine::Clients)
```

Comment:
I reckon this is an implementation detail of `DiscoveryEngine::Evaluation::SampleQuerySet`, we probably shouldn't need to stub that here.